index.html

```diff
@@ -5004,13 +5004,12 @@ <h2 id="AudioParam">
           </li>
         </ul>
         <p class="note">
-          <code>AudioParam</code>s are read only and must be set using the
-          <a>automation methods</a> or the <a href=
-          "#widl-AudioParam-value">value</a> attribute. Using the <a href=
-          "#widl-AudioParam-value">value</a> attribute is equivalent to calling
-          <code>setValueAtTime()</code> with the current
-          <code>AudioContext</code>'s <code>currentTime</code> and the
-          requested value.
+          <code>AudioParam</code> attributes are read only, with the exception
+          of the <a href="#dom-audioparam-value">value</a> attribute. Setting
+          the <a href="#dom-audioparam-value">value</a> attribute is
```

Comment:
Maybe leave out the "Setting the value..." sentence? It's not really a note any more in light of the more explicit explanation below.

Reply:
Makes sense to me - I'll remove it. This duplication already existed so wasn't sure what to do.